bindata/network/ovn-kubernetes/common/008-script-lib.yaml

```diff
@@ -332,6 +332,11 @@ data:
         ipsec_encapsulation=true
       fi
 {{ end }}
+
+{{ if eq .OVNIPsecEncap "Always" }}
+      ipsec_encapsulation=true
```

Comment:
isnt it possible this key-value is defined twice if on ibm platform and IPSec Full option set to Always?

Reply:
shouldnt this be within the OVNIPsecEnable if block? I know your programming is fine here but it reads weird and id like checking `OVNIPsecEncap` eqs Always to be inside the `OVNIPsecEnable` bool.